src/client/js/Utils/SaveToDisk.js

```diff
@@ -0,0 +1,34 @@
+/*globals define*/
+/*jshint node: true, browser: true, bitwise: false*/
+
+/**
+ * @author kecso / https://github.com/kecso
+ */
+define(function () {
+    'use strict';
+
+    var saveToDisk = function (fileURL, fileName) {
```

Comment:
Was this code snippet coming from some external source, e.g. stack overflow? If so it's good to add a link.